MatrixKit/Utils/MXKSoundPlayer.m

```diff
@@ -74,9 +74,6 @@ - (void)stopPlaying
     {
         [self.audioPlayer stop];
         self.audioPlayer = nil;
-        
-        // Release the audio session to allow resuming of background music app
-        [[AVAudioSession sharedInstance] setActive:NO withOptions:AVAudioSessionSetActiveOptionNotifyOthersOnDeactivation error:nil];
```

Comment:
ATM, I am more for a parameter because AVAudioSession management will stay in MXKSoundPlayer.
MXCallVC is then only a relay between MXCallKitAdapter and MXKSoundPlayer.